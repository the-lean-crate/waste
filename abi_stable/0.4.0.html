<!DOCTYPE html><html><head><title>abi_stable:0.4.0</title><span style="position: fixed; top: 1em; right: 1em; color: pink">Ugly Alpha 1</span></head><body><article><title>abi_stable:0.4.0</title><section id="total-bytes"><h3>total uncompressed bytes</h3><p>792.2 KB</p></section><section id="total-files"><h3>total files</h3><p>108</p></section><section id="potential-savings"><h3>potential savings</h3><p>82.3 KB total in 10 files</p></section><section><h3>Fix</h3><section>ImprovedInclude {
    include: [
        &quot;src/**/*.rs&quot;,
        &quot;../readme.md&quot;,
        &quot;../APACHE_license&quot;,
        &quot;../MIT_license&quot;,
    ],
    include_removed: [
        &quot;Cargo.toml&quot;,
    ],
    potential: Some(
        PotentialWaste {
            patterns_to_fix: [
                &quot;!**/docs/*&quot;,
                &quot;!**/tests/*&quot;,
                &quot;!**/*_tests.*&quot;,
                &quot;!**/*_tests/*&quot;,
            ],
            potential_waste: [
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        105,
                        108,
                        105,
                        116,
                        121,
                        47,
                        108,
                        97,
                        121,
                        111,
                        117,
                        116,
                        95,
                        116,
                        101,
                        115,
                        116,
                        115,
                        47,
                        101,
                        114,
                        97,
                        115,
                        101,
                        100,
                        95,
                        116,
                        121,
                        112,
                        101,
                        115,
                        46,
                        114,
                        115,
                    ],
                    size: 2501,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        105,
                        108,
                        105,
                        116,
                        121,
                        47,
                        108,
                        97,
                        121,
                        111,
                        117,
                        116,
                        95,
                        116,
                        101,
                        115,
                        116,
                        115,
                        47,
                        112,
                        114,
                        101,
                        102,
                        105,
                        120,
                        95,
                        116,
                        121,
                        112,
                        101,
                        115,
                        46,
                        114,
                        115,
                    ],
                    size: 29957,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        105,
                        108,
                        105,
                        116,
                        121,
                        47,
                        108,
                        97,
                        121,
                        111,
                        117,
                        116,
                        95,
                        116,
                        101,
                        115,
                        116,
                        115,
                        47,
                        114,
                        101,
                        112,
                        114,
                        95,
                        97,
                        110,
                        100,
                        95,
                        100,
                        105,
                        115,
                        99,
                        114,
                        46,
                        114,
                        115,
                    ],
                    size: 5877,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        105,
                        108,
                        105,
                        116,
                        121,
                        47,
                        108,
                        97,
                        121,
                        111,
                        117,
                        116,
                        95,
                        116,
                        101,
                        115,
                        116,
                        115,
                        47,
                        118,
                        97,
                        108,
                        117,
                        101,
                        46,
                        114,
                        115,
                    ],
                    size: 21359,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        105,
                        108,
                        105,
                        116,
                        121,
                        47,
                        108,
                        97,
                        121,
                        111,
                        117,
                        116,
                        95,
                        116,
                        101,
                        115,
                        116,
                        115,
                        46,
                        114,
                        115,
                    ],
                    size: 258,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        100,
                        111,
                        99,
                        115,
                        47,
                        112,
                        114,
                        101,
                        102,
                        105,
                        120,
                        95,
                        116,
                        121,
                        112,
                        101,
                        115,
                        46,
                        114,
                        115,
                    ],
                    size: 7726,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        100,
                        111,
                        99,
                        115,
                        47,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        95,
                        97,
                        98,
                        105,
                        95,
                        100,
                        101,
                        114,
                        105,
                        118,
                        101,
                        46,
                        114,
                        115,
                    ],
                    size: 6112,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        100,
                        111,
                        99,
                        115,
                        47,
                        117,
                        110,
                        115,
                        97,
                        102,
                        101,
                        95,
                        99,
                        111,
                        100,
                        101,
                        95,
                        103,
                        117,
                        105,
                        100,
                        101,
                        108,
                        105,
                        110,
                        101,
                        115,
                        46,
                        114,
                        115,
                    ],
                    size: 1251,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        109,
                        105,
                        115,
                        99,
                        95,
                        116,
                        101,
                        115,
                        116,
                        115,
                        46,
                        114,
                        115,
                    ],
                    size: 56,
                    entry_type: 48,
                },
                TarHeader {
                    path: [
                        97,
                        98,
                        105,
                        95,
                        115,
                        116,
                        97,
                        98,
                        108,
                        101,
                        45,
                        48,
                        46,
                        52,
                        46,
                        48,
                        47,
                        115,
                        114,
                        99,
                        47,
                        114,
                        101,
                        102,
                        108,
                        101,
                        99,
                        116,
                        105,
                        111,
                        110,
                        47,
                        116,
                        101,
                        115,
                        116,
                        115,
                        47,
                        100,
                        101,
                        114,
                        105,
                        118,
                        101,
                        95,
                        114,
                        101,
                        102,
                        108,
                        101,
                        99,
                        116,
                        105,
                        111,
                        110,
                        46,
                        114,
                        115,
                    ],
                    size: 7157,
                    entry_type: 48,
                },
            ],
        },
    ),
    has_build_script: false,
}</section></section></article></body><footer><span>Created by <a href="https://github.com/Byron/">Byron</a></span> | <span><a href="https://github.com/the-lean-crate/criner/issues/new?labels=bug&amp;template=waste-report-incorrect.md&amp;title=%5BINCORRECT+WASTE+REPORT%5D+%3CCrate+Name%3E" target="_blank" rel="noopener noreferrer">Provide feedback</a></span> | <span>Generated by <a href="https://github.com/the-lean-crate/criner#the-criner-waste-report">Criner</a> at 2020-05-18T06:58:07Z</span></footer></html>