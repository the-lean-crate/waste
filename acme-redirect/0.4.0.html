<!DOCTYPE html><html><head><title>acme-redirect:0.4.0</title><span style="position: fixed; top: 1em; right: 1em; color: pink">Ugly Alpha 1</span></head><body><article><title>acme-redirect:0.4.0</title><section id="total-bytes"><h3>total uncompressed bytes</h3><p>177.9 KB</p></section><section id="total-files"><h3>total files</h3><p>44</p></section><section><h3>Fix</h3><section>NewInclude {
    include: [
        &quot;src/**/*&quot;,
        &quot;Cargo.*&quot;,
        &quot;LICENSE&quot;,
        &quot;README.md&quot;,
    ],
    has_build_script: false,
}</section></section><section><h3>24 wasted files</h3><p>total waste: 11.3 KB</p><ol><li>PKGBUILD : 2.0 KB</li><li>contrib/docs/acme-redirect.1.scd : 2.0 KB</li><li>.github/workflows/release.yml : 1.1 KB</li><li>contrib/docs/acme-redirect.conf.5.scd : 1.0 KB</li><li>contrib/docs/acme-redirect.d.5.scd : 896 B</li><li>examples/boxxy.rs : 610 B</li><li>.github/workflows/debian.yml : 544 B</li><li>contrib/openrc/acme-redirect.initd : 496 B</li><li>Dockerfile : 404 B</li><li>.github/workflows/docker.yml : 340 B</li><li>contrib/systemd/acme-redirect.service : 328 B</li><li>.github/workflows/rust.yml : 283 B</li><li>contrib/systemd/acme-redirect.tmpfiles : 176 B</li><li>contrib/cron/acme-redirect.cron : 175 B</li><li>Makefile : 167 B</li><li>contrib/systemd/acme-redirect-renew.timer : 136 B</li><li>contrib/confs/certs.d/example.com.conf : 127 B</li><li>contrib/confs/acme-redirect.conf : 110 B</li><li>contrib/systemd/acme-redirect-renew.service : 109 B</li><li>contrib/systemd/acme-redirect.sysusers : 61 B</li><li>.dockerignore : 58 B</li><li>debian/postinst : 53 B</li><li>.gitignore : 49 B</li><li>.github/FUNDING.yml : 17 B</li></ol></section></article></body><footer><span>Created by <a href="https://github.com/Byron/">Byron</a></span> | <span><a href="https://github.com/the-lean-crate/criner/issues/new?labels=bug&amp;template=waste-report-incorrect.md&amp;title=%5BINCORRECT+WASTE+REPORT%5D+%3CCrate+Name%3E" target="_blank" rel="noopener noreferrer">Provide feedback</a></span> | <span>Generated by <a href="https://github.com/the-lean-crate/criner#the-criner-waste-report">Criner</a> at 2020-05-24T23:56:42Z</span></footer></html>