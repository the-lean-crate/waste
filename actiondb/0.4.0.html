<!DOCTYPE html><html><head><title>actiondb:0.4.0</title><span style="position: fixed; top: 1em; right: 1em; color: pink">Ugly Alpha 1</span></head><body><article><title>actiondb:0.4.0</title><section id="total-bytes"><h3>total uncompressed bytes</h3><p>52.7 MB</p></section><section id="total-files"><h3>total files</h3><p>91</p></section><section><h3>Fix</h3><section>NewInclude {
    include: [
        &quot;src/**/*&quot;,
        &quot;README.md&quot;,
        &quot;CHANGELOG.md&quot;,
    ],
    has_build_script: false,
}</section></section><section><h3>33 wasted files</h3><p>total waste: 52.5 MB</p><ol><li>ssh.txt : 50.7 MB</li><li>perf.data.old : 740.5 KB</li><li>test3.json : 606.0 KB</li><li>perf.data : 186.6 KB</li><li>tags : 158.2 KB</li><li>.tags : 10.1 KB</li><li>pdb2adb : 9.3 KB</li><li>utils/pdb2adb : 9.3 KB</li><li>test.json : 4.1 KB</li><li>tests/matcher/mod.rs : 3.7 KB</li><li>tests/file/mod.rs : 1.4 KB</li><li>tests/matcher/ssh_ok.json : 1.3 KB</li><li>tests/matcher/ssh_tags_are_not_there.json : 1.3 KB</li><li>tests/matcher/ssh_only_expected_tags_are_checked.json : 1.3 KB</li><li>ssh_wrong_uuid.json : 1.3 KB</li><li>tests/file/ssh_ok.json : 1.1 KB</li><li>test2.json : 832 B</li><li>tests/matcher/pattern_validation_should_not_be_local.json~ : 785 B</li><li>tests/matcher/pattern_validation_should_not_be_local.json : 723 B</li><li>tests/loggen.json : 705 B</li><li>utils/README.md : 542 B</li><li>tests/matcher/ssh_only_expected_values_are_checked.json : 525 B</li><li>tests/matcher/ssh_when_an_expected_value_is_not_found_we_fail.json : 508 B</li><li>tests/matcher/ssh_we_can_parse_multiline_messages.json : 435 B</li><li>.travis.yml : 402 B</li><li>ssh.txt~ : 387 B</li><li>.tags1 : 344 B</li><li>tests/matcher/ssh_wrong.json : 314 B</li><li>tests/file/ssh_wrong.json : 314 B</li><li>tests/ssh_log.txt : 287 B</li><li>.gdb_history : 92 B</li><li>.gitignore : 91 B</li><li>tests/lib.rs : 47 B</li></ol></section></article></body><footer><span>Created by <a href="https://github.com/Byron/">Byron</a></span> | <span><a href="https://github.com/the-lean-crate/criner/issues/new?labels=bug&amp;template=waste-report-incorrect.md&amp;title=%5BINCORRECT+WASTE+REPORT%5D+%3CCrate+Name%3E" target="_blank" rel="noopener noreferrer">Provide feedback</a></span> | <span>Generated by <a href="https://github.com/the-lean-crate/criner#the-criner-waste-report">Criner</a> at 2020-05-18T06:58:12Z</span></footer></html>