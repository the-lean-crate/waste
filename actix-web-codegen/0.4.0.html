<!DOCTYPE html><html><head><title>actix-web-codegen:0.4.0</title><span style="position: fixed; top: 1em; right: 1em; color: pink">Ugly Alpha 1</span></head><body><article><title>actix-web-codegen:0.4.0</title><section id="total-bytes"><h3>total uncompressed bytes</h3><p>46.0 KB</p></section><section id="total-files"><h3>total files</h3><p>21</p></section><section><h3>Fix</h3><section>NewInclude {
    include: [
        &quot;src/**/*&quot;,
        &quot;LICENSE-*&quot;,
        &quot;README.md&quot;,
    ],
    has_build_script: false,
}</section></section><section><h3>15 wasted files</h3><p>total waste: 15.4 KB</p><ol><li>tests/test_macro.rs : 7.8 KB</li><li>CHANGES.md : 1.5 KB</li><li>tests/trybuild.rs : 815 B</li><li>tests/trybuild/simple-fail.stderr : 676 B</li><li>tests/trybuild/route-missing-method-fail.stderr : 526 B</li><li>tests/trybuild/simple-fail.rs : 499 B</li><li>tests/trybuild/route-duplicate-method-fail.stderr : 442 B</li><li>tests/trybuild/route-unexpected-method-fail.stderr : 425 B</li><li>tests/trybuild/route-missing-method-fail-msrv.stderr : 415 B</li><li>tests/trybuild/simple.rs : 393 B</li><li>tests/trybuild/route-ok.rs : 389 B</li><li>tests/trybuild/route-duplicate-method-fail.rs : 388 B</li><li>tests/trybuild/route-unexpected-method-fail.rs : 381 B</li><li>tests/trybuild/route-missing-method-fail.rs : 364 B</li><li>tests/trybuild/route-missing-method-fail-msrv.rs : 364 B</li></ol></section></article></body><footer><span>Created by <a href="https://github.com/Byron/">Byron</a></span> | <span><a href="https://github.com/the-lean-crate/criner/issues/new?labels=bug&amp;template=waste-report-incorrect.md&amp;title=%5BINCORRECT+WASTE+REPORT%5D+%3CCrate+Name%3E" target="_blank" rel="noopener noreferrer">Provide feedback</a></span> | <span>Generated by <a href="https://github.com/the-lean-crate/criner#the-criner-waste-report">Criner</a> at 2020-09-25T04:09:06Z</span></footer></html>