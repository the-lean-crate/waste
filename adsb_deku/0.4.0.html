<!DOCTYPE html><html><head><title>adsb_deku:0.4.0</title><span style="position: fixed; top: 1em; right: 1em; color: pink">Ugly Alpha 1</span></head><body><article><title>adsb_deku:0.4.0</title><section id="total-bytes"><h3>total uncompressed bytes</h3><p>6.7 MB</p></section><section id="total-files"><h3>total files</h3><p>24</p></section><section><h3>Fix</h3><section>NewInclude {
    include: [
        &quot;src/**/*&quot;,
        &quot;LICENSE-MIT&quot;,
        &quot;README.md&quot;,
        &quot;CHANGELOG.md&quot;,
    ],
    has_build_script: false,
}</section></section><section><h3>14 wasted files</h3><p>total waste: 6.6 MB</p><ol><li>tests/lax-messages.txt : 4.7 MB</li><li>media/peek_2021_10_31.gif : 847.8 KB</li><li>media/2021_12_08.gif : 825.9 KB</li><li>media/2021-10-31-093905_676x659_scrot.png : 67.1 KB</li><li>media/logo_400_400.png : 52.2 KB</li><li>media/2021-10-31-094541_1080x548_scrot.png : 50.3 KB</li><li>tests/test.rs : 28.4 KB</li><li>media/logo.png : 24.6 KB</li><li>media/logo.xcf : 17.5 KB</li><li>benches/decoding.rs : 2.1 KB</li><li>.github/workflows/main.yml : 1.5 KB</li><li>tests/file.rs : 747 B</li><li>.rustfmt.toml : 228 B</li><li>.gitignore : 19 B</li></ol></section></article></body><footer><span>Created by <a href="https://github.com/Byron/">Byron</a></span> | <span><a href="https://github.com/the-lean-crate/criner/issues/new?labels=bug&amp;template=waste-report-incorrect.md&amp;title=%5BINCORRECT+WASTE+REPORT%5D+%3CCrate+Name%3E" target="_blank" rel="noopener noreferrer">Provide feedback</a></span> | <span>Generated by <a href="https://github.com/the-lean-crate/criner#the-criner-waste-report">Criner</a> at 2021-12-09T23:31:11Z</span></footer></html>