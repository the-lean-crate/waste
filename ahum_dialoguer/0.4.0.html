<!DOCTYPE html><html><head><title>ahum_dialoguer:0.4.0</title><span style="position: fixed; top: 1em; right: 1em; color: pink">Ugly Alpha 1</span></head><body><article><title>ahum_dialoguer:0.4.0</title><section id="total-bytes"><h3>total uncompressed bytes</h3><p>60.2 KB</p></section><section id="total-files"><h3>total files</h3><p>20</p></section><section><h3>Fix</h3><section>NewInclude {
    include: [
        &quot;src/**/*&quot;,
        &quot;LICENSE&quot;,
        &quot;README.md&quot;,
    ],
    has_build_script: false,
}</section></section><section><h3>11 wasted files</h3><p>total waste: 6.1 KB</p><ol><li>examples/wizard.rs : 2.2 KB</li><li>examples/checkboxes.rs : 683 B</li><li>examples/select_opt.rs : 617 B</li><li>examples/select.rs : 500 B</li><li>examples/continue.rs : 451 B</li><li>examples/file_prompt.rs : 444 B</li><li>examples/password.rs : 400 B</li><li>examples/prompt_character.rs : 320 B</li><li>examples/edit.rs : 261 B</li><li>.travis.yml : 216 B</li><li>.gitignore : 18 B</li></ol></section></article></body><footer><span>Created by <a href="https://github.com/Byron/">Byron</a></span> | <span><a href="https://github.com/the-lean-crate/criner/issues/new?labels=bug&amp;template=waste-report-incorrect.md&amp;title=%5BINCORRECT+WASTE+REPORT%5D+%3CCrate+Name%3E" target="_blank" rel="noopener noreferrer">Provide feedback</a></span> | <span>Generated by <a href="https://github.com/the-lean-crate/criner#the-criner-waste-report">Criner</a> at 2020-05-18T06:58:07Z</span></footer></html>