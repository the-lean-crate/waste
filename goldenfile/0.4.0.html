<!DOCTYPE html><html><head><title>goldenfile:0.4.0</title><span style="position: fixed; top: 1em; right: 1em; color: pink">Ugly Alpha 1</span></head><body><article><title>goldenfile:0.4.0</title><section id="total-bytes"><h3>total uncompressed bytes</h3><p>12.5 KB</p></section><section id="total-files"><h3>total files</h3><p>16</p></section><section><h3>Fix</h3><section>NewInclude {
    include: [
        &quot;src/**/*&quot;,
        &quot;LICENSE&quot;,
        &quot;README.md&quot;,
    ],
    has_build_script: false,
}</section></section><section><h3>10 wasted files</h3><p>total waste: 3.5 KB</p><ol><li>tests/basics.rs : 1.9 KB</li><li>tests/usage.rs : 689 B</li><li>scripts/presubmit : 522 B</li><li>scripts/install-git-hooks : 160 B</li><li>.travis.yml : 99 B</li><li>scripts/git-hooks/pre-commit : 66 B</li><li>.gitignore : 23 B</li><li>tests/.gitignore : 13 B</li><li>tests/goldenfiles/file1.txt : 12 B</li><li>tests/goldenfiles/file2.txt : 6 B</li></ol></section></article></body><footer><span>Created by <a href="https://github.com/Byron/">Byron</a></span> | <span><a href="https://github.com/the-lean-crate/criner/issues/new?labels=bug&amp;template=waste-report-incorrect.md&amp;title=%5BINCORRECT+WASTE+REPORT%5D+%3CCrate+Name%3E" target="_blank" rel="noopener noreferrer">Provide feedback</a></span> | <span>Generated by <a href="https://github.com/the-lean-crate/criner#the-criner-waste-report">Criner</a> at 2020-05-18T06:59:19Z</span></footer></html>